src/vstest.console/vstest.console.csproj

```diff
@@ -20,6 +20,9 @@
   <ItemGroup>
     <EmbeddedResource Include="Resources\Resources.resx" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="1.1.1" />
```

Comment:
Try move this to the TestPlatform.Depependencies.props

Reply:
Done